client/components/payment-logo/index.jsx

```diff
@@ -31,6 +31,7 @@ const ALT_TEXT = {
 	wechat: i18n.translate( 'WeChat Pay', {
 		comment: 'Name for WeChat Pay - https://pay.weixin.qq.com/',
 	} ),
+	'web-payment': 'Web Payment',
```

Comment:
This is an Apple product name that should not be translated, correct?

Reply:
It's not the actual product, and we'll probably use something different when we launch. Just a placeholder for now :)